### ISSUE TITLE
Dashboards for Forest website

### ISSUE THREAD

**Issue by LesnyRumcajs** [ChainSafe/forest-iac#231]
**Issue summary**

A little bit of background - we plan to ship a Forest website later this year (see [Forest Roadmap 2023](https://github.com/ChainSafe/forest/issues/1874)). There, we plan to embed some dashboards that will show the status of our long-running nodes.

The dashboards on the website should be in the form of a summary for slightly-but-not-overly-technical people. That is to say; they should expect a small level of technical competence (e.g., understanding the difference between CPU and memory usage) and Filecoin arcana knowledge (epochs, peers), but not details (like the number of ChainExchange p2p messages).

At the moment, we have some basic New Relic [dashboards](https://github.com/ChainSafe/forest-iac/blob/main/terraform/new-relic/forest.json) migrated from [Grafana](https://github.com/ChainSafe/forest/blob/main/.maintain/monitoring/grafana/dashboards/forest.json). They are a good start, but we need more structure.

Beforehand, we must consider whether we can easily embed the dashboards in New Relic or Grafana on the website. If NR is not possible (to my understanding, it should be fine - given a web application, we could use NR API to request a PDF of the dashboard and embed it in the website), we can consider opening a Grafana endpoint (which would require additional work). Alternatively, the web application could manually generate the dashboards using raw NR API data, which would put more effort on the team responsible for the website.

### Tasks
- [x] Investigate if and how NR offers to make dashboards public. Summarize the findings in this thread.
- [x] If the NR investigation proves that it's not feasible to do with NR, consider local Grafana (or Grafana Cloud?),
- [ ] Discuss with the team on the best approach,
- [x] #236
  - Head epoch,
  - Tipsets validated per minute (this should be always ~2),
  - Number of connected peers,
  - Process uptime (single number)
  - CPU usage,
  - Memory usage (Process Resident Memory and NOT virtual memory),
  - Disk usage
All metrics should be absolute, i.e., disk usage should not show 50% because it means nothing without knowledge of the disk size.
- [ ] Additional pages with more detailed metrics. This is not directly related to the website functionality, so we can do it later. For now, we can just put the current metrics there.
- [ ] Ensure that the results on every chart make sense, both in shape and value,
- [ ] Present the dashboard to the team; @lerajk acceptance is required. Implement requested changes.
- [x] #238
- [ ] Prepare a short guide for the website team on integrating the dashboard into the website.

### Acceptance criteria
- [ ] Forest and Lotus dashboards are publicly available,
- [x] #239
- [ ] Dashboards are structured, with the first page/tab offering an executive summary of the node.
- [ ] A guide for the website team is ready.

**Other information and links**

Feel free to split this issue into subtasks. Some can be worked on in parallel.

**Comment by LesnyRumcajs** [comment]
Stale, at the moment `won't do`.
